sdk/go/pulumi/types.go

```diff
@@ -1217,3 +1219,67 @@ func init() {
 	RegisterOutputType(ResourceOutput{})
 	RegisterOutputType(ResourceArrayOutput{})
 }
+
```

Comment:
Personally, I would remove this stack trace comment and adjust line 289 to indicate how to reproduce the stack trace. I think that's preferable to including a trace that could become stale and grabs a lot of screen real estate.

Reply:
+1